src/tokens/transformation/tailwind/formatTokens.js

```diff
@@ -98,27 +95,25 @@ function createColorTokensFromGroup(tokens, staticTheme = true) {
         colorGroup = formatColorVar(name, staticTheme)
       }
 
-      colorTokens[t.attributes.type] = colorGroup
+      if (Object.keys(colorGroup).length > 0) {
+        colorTokens[t.attributes.type] = colorGroup
+      }
     }
   })
   return colorTokens
 }
 
-function createStaticColorTokens(tokens) {
-  return createColorTokensFromGroup(tokens)
-}
-
 function createDynamicColorTokens(tokens) {
   return createColorTokensFromGroup(tokens, false)
 }
 
 module.exports = ({ dictionary }) => {
-  const staticColorTokens = createStaticColorTokens(dictionary.allTokens)
   const dynamicColorTokens = createDynamicColorTokens(dictionary.allTokens)
-  const colorTokens = merge(dynamicColorTokens, staticColorTokens)
+  const colorTokens = merge(dynamicColorTokens)
 
   const fontSizes = new Map()
   const borderRadii = new Map()
+  const spacing = new Map([[0, 0]]) // Initialize with option for 0 spacing
```

Comment:
This is so we can do something like:

```html
<div class="p-m pt-0">
```

right? Do we need to do anything special to make sure tailwind knows that we want to be able to override them?

Reply:
Additionally, should we also have a `0` option for `borderRadii`?

Reply:
> right?

Correct.

> Do we need to do anything special to make sure tailwind knows that we want to be able to override them?

Can you clarify?

> Additionally, should we also have a 0 option for borderRadii?

No, tailwind has a `border-none` class.

Reply:
So I can still use `rounded-none` to set the border radius (only) to zero?

Reply:
Shoot. No sorry. I was thinking `border` not `rounded`. My bad.

Reply:
Resolved.